contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -1182,7 +1181,7 @@ END;
 $$ LANGUAGE plpgsql IMMUTABLE STRICT;
 
 CREATE OR REPLACE VIEW sys.systypes AS
```

Comment:
Definition in upgrade is different from this one. Please make sure if we are making some changes in later commit, it should reflect in upgrade as well.

Reply:
done